### ISSUE TITLE
localstorage support

### ISSUE THREAD

**Issue by hexxt-git** [Andiedie/sync-my-cookie#27]
not all websites save user data on cookies there is localsorage and sessionstorage. i am specifically interested about this to sync my excalidraw board

**Comment by AJolly** [comment]
Yeah this would be awesome, but it doesnt look like this extensions updated at all.

**Comment by Walkoud** [comment]
interested too

**Comment by guotao** [comment]
可以试试 [SyncMeIn](https://kainy.cn/SyncMeIn/#gh) 支持localstorage同步，免费账户有同步的域名数量限制就是了。

**Comment by hexxt-git** [comment]
> 可以试试 [SyncMeIn](https://kainy.cn/SyncMeIn/#gh) 支持localstorage同步，免费账户有同步的域名数量限制就是了。

honestly i don't trust it.
this extension sync my cookies uses github to store your data and is open source

**Comment by AJolly** [comment]
@hexxt-git yeah, and SyncMeIn has copied most of the source code straight from this project.

**Comment by guotao** [comment]
> > 可以试试 [SyncMeIn](https://kainy.cn/SyncMeIn/#gh) 支持localstorage同步，免费账户有同步的域名数量限制就是了。
> 
> honestly i don't trust it. this extension sync my cookies uses github to store your data and is open source

嗯，我也是选择性的用于，对便利性要求高于安全性的场景，比如公众平台自动化以及公司内网系统，外部根本无法访问。
